hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/TransactionProcessorTest.java

```diff
@@ -16,13 +16,43 @@
 
 package com.hedera.node.app.service.contract.impl.test.exec;
```

Comment:
Code coverage shows missed L223 in `TransactionProcessor.computeInvolvedParties` ([equivalent link](https://github.com/hashgraph/hedera-services/blob/0f10af45473ac79c98b609fc180dae8c659daa70/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java#L203)) which looks like it might be interesting to cover.